lib/external-grading/file-store-s3.js

```diff
@@ -0,0 +1,70 @@
+const ERR = require('async-stacktrace');
+const async = require('async');
+const tar = require('tar');
+const AWS = require('aws-sdk');
+const tmp = require('tmp');
+const sqldb = require('@prairielearn/prairielib/sql-db');
+const sqlLoader = require('@prairielearn/prairielib/sql-loader');
+
+const config = require('../config');
+const externalGraderCommon = require('./common');
+
+const sql = sqlLoader.loadSqlEquiv(__filename);
```

Comment:
Is there a `file-store-s3.sql` included with this PR?

Reply:
Very good catch! I'll move that over from the appropriate old file.